sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -579,39 +391,276 @@
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if calculatedDataChildCareCost &#62; 0&#10;  then min(((offeringCourseLoad/100) * dmnPartTimeProgramYearMaximums.limitWeeklyChildCare &#10;    * 17 * calculatedDataTotalEligibleDependentsForChildCare),&#10;  calculatedDataChildCareCost)&#10;else 0" target="calculatedDataTotalChildCareCost" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1v61q2r</bpmn:incoming>
+      <bpmn:outgoing>Flow_19bk6l4</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:exclusiveGateway id="Gateway_1m98y7x">
       <bpmn:incoming>Flow_0temot0</bpmn:incoming>
-      <bpmn:outgoing>Flow_0lq5uy6</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0djv5vn</bpmn:outgoing>
+      <bpmn:outgoing>Flow_18af5gw</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1v61q2r</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:exclusiveGateway id="Gateway_0r706cf">
+      <bpmn:incoming>Flow_16xshp2</bpmn:incoming>
+      <bpmn:incoming>Flow_0iqqipy</bpmn:incoming>
+      <bpmn:incoming>Flow_19bk6l4</bpmn:incoming>
+      <bpmn:outgoing>Flow_0jaycgo</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_0hxxmu9" name="Define Total Income">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if&#10;  partner1CRAReportedIncome = null&#10;then&#10;  partner1TotalIncome&#10;else&#10;  partner1CRAReportedIncome" target="calculatedDataPartner1TotalIncome" />
+          <zeebe:output source="=if&#10;  studentDataCRAReportedIncome = null&#10;then&#10;  studentDataTaxReturnIncome&#10;else&#10;  studentDataCRAReportedIncome" target="calculatedDataStudentTotalIncome" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1idr6ms</bpmn:incoming>
+      <bpmn:outgoing>Flow_0dqxcli</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_18l6de5" name="Calculate Eligible Dependents">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if studentDataDependants = null then 0 else&#10;  count(&#10;studentDataDependants[&#10;// 0-18 years of age: eligible.&#10;date(item.dateOfBirth) &#62;= (date(offeringStudyStartDate) - duration(&#34;P18Y&#34;))&#10;// 19-22 years of age: post-sec must be Yes or Tax is Yes.&#10;or (&#10;  date(item.dateOfBirth) &#60; (date(offeringStudyStartDate) - duration(&#34;P18Y&#34;))&#10;  and date(item.dateOfBirth) &#62;= (date(offeringStudyStartDate) - duration(&#34;P22Y&#34;))&#10;  and (item.attendingPostSecondarySchool = &#34;yes&#34; or item.declaredOnTaxes = &#34;yes&#34;)&#10;)&#10;// &#62; 22 years: eligible if Tax is Yes.&#10;or (&#10;  date(item.dateOfBirth) &#60; (date(offeringStudyStartDate) - duration(&#34;P22Y&#34;))&#10;  and item.declaredOnTaxes = &#34;yes&#34;&#10;)&#10;]&#10;  )" target="calculatedDataTotalEligibleDependants" />
```

Comment:
I do recognize that we do not have a format for the Feel Expression but, as a suggestion, I would like to preserve some level of indentation as below.
```
if studentDataDependants = null then 0
else
  count(
    studentDataDependants[
      // 0-18 years of age: eligible.
      date(item.dateOfBirth) >= (date(offeringStudyStartDate) - duration("P18Y"))
      // 19-22 years of age: post-sec must be Yes or Tax is Yes.
      or (
        date(item.dateOfBirth) < (date(offeringStudyStartDate) - duration("P18Y"))
        and date(item.dateOfBirth) >= (date(offeringStudyStartDate) - duration("P22Y"))
        and (item.attendingPostSecondarySchool = "yes" or item.declaredOnTaxes = "yes")
      )
      // > 22 years: eligible if Tax is Yes.
      or (
        date(item.dateOfBirth) < (date(offeringStudyStartDate) - duration("P22Y"))
        and item.declaredOnTaxes = "yes")
    ]
  )
```